<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindWidget</class>
 <widget class="QWidget" name="FindWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>46</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="hideButton">
     <property name="icon">
      <iconset theme="dialog-close">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="findText">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nextButton">
     <property name="toolTip">
      <string>Jump to next match</string>
     </property>
     <property name="icon">
      <iconset theme="go-down-search">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="previousButton">
     <property name="toolTip">
      <string>Jump to previous match</string>
     </property>
     <property name="icon">
      <iconset theme="go-up-search">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="matchCase">
     <property name="toolTip">
      <string>Match case sensitive</string>
     </property>
     <property name="icon">
      <iconset theme="format-text-superscript">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hideButton</sender>
   <signal>clicked()</signal>
   <receiver>FindWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
